app/controllers/recipe_revisions_controller.rb

```diff
@@ -4,27 +4,28 @@ class RecipeRevisionsController < ApplicationController
 
   def index
     @user             = @recipe.user
-    @revisions        = RecipeRevision.where(:recipe_id => @recipe.id)
+    @revisions        = RecipeRevision.where(recipe_id: @recipe.id)
     @recipe_permalink = @recipe.slug
   end
 
   def show
-    revision = RecipeRevision.find(params[:id])
+    @revision = RecipeRevision.find(params[:id])
 
-    if revision.revision > 1
-      previous_revision = RecipeRevision.where(:recipe_id => revision.recipe_id,
-                                               :revision => revision.revision - 1).first
-
-      @diff = Differ.diff_by_line(revision.body, previous_revision.body).format_as(:html).html_safe
+    if params[:target_id].present?
```

Comment:
Could you change target_id to something with a bit more semantic meaning? Maybe something along the lines of forked_recipe_revsion.